node-yun-http/platformio.ini

```diff
@@ -0,0 +1,27 @@
+; PlatformIO Project Configuration File for Mois sensor node
+;
+; Please visit documentation for the other options and examples
+; https://docs.platformio.org/en/latest/projectconf/
+
+[platformio]
+src_dir = .
+
+monitor_speed = 115200
+
+[common]
+lib_deps =
+    ../libraries/ADS1231
+    adafruit/Adafruit TSL2591 Library@1.4.3
+    adafruit/DHT sensor library@1.4.4
+    arduino-libraries/Bridge@1.7.0
+    milesburton/DallasTemperature@3.11.0
+    paulstoffregen/OneWire@2.3.5
+    robtillaart/RunningMedian@0.3.7
+    watterott/digitalWriteFast@1.0.0
+
+[env:atmega328p]
+platform = atmelavr
+board = pro8MHzatmega328
+framework = arduino
```

Comment:
Can you confirm this selects the right AVR chip on the Yun?

Reply:
das muss ich mir morgen genauer anschauen.

Reply:
It may actually be an ATmega32u4. I will try to find the correct board identifier for PlatformIO.

Reply:
wo finde ich die infos?

ich arbeite mit folgenden board-einstellungen in der arduino-ide:

- arduino uno - dragino yun (jeden falls NICHT auf Arduino Yún!!)
- USBtinyISP
- (w)lan-verbindung herstellen ins gleiche lan: ip unter "port" auswählen

Reply:
All right. `pro8MHzatmega328` is probably the right choice then.

Reply:
Ah. I've been told to just use `uno` as a board identifier. _That_ is the right thing, running on 16 Mhz in this case.

-- https://docs.platformio.org/en/latest/boards/atmelavr/uno.html

Reply:
sounds reasonable (though I do not completely understand what I am doing here as long as it works for me. so I gladly trust your deeper understanding of the matter. i will yell if something will be broken for me and i cant understand why).

Reply:
> ich arbeite mit folgenden board-einstellungen in der arduino-ide:
> 
>     * arduino uno - dragino yun (jeden falls NICHT auf Arduino Yún!!)

Was verwendest du denn als Hardware, das Dragino-Yun-shield oder den Arduino Yun? Und wenn du das shield, welches board verwendest du als base? s. auch https://community.platformio.org/t/it-posible-to-add-dragino-board/10208/6

Reply:
I will be using `board = uno` now. Merging.